contract_price_revision/tests/test_contract_price_revision.py

```diff
@@ -82,3 +82,28 @@ def test_contract_price_revision_invoicing(self):
         self.assertEqual(len(lines), 2)
         lines = lines.filtered(lambda x: x.price_unit == 200.0)
         self.assertEqual(len(lines), 1)
+
+    def test_compute_variation_percent(self):
+        self.acct_line.price_unit = 200.0
+        self.acct_line.previous_price = 100.0
+        self.acct_line._compute_variation_percent()
+        self.assertEqual(self.acct_line.variation_percent, 100.0)
+        self.acct_line.price_unit = 200.0
+        self.acct_line.previous_price = 0.0
+        self.acct_line._compute_variation_percent()
+        self.assertEqual(self.acct_line.variation_percent, 0.0)
+
+    def test_get_default_variation_type(self):
+        variation_type = self.env[
+            "contract.price.revision.wizard"
+        ]._get_default_variation_type()
+        self.assertEqual(variation_type, "percentage")
+
+    def test_get_new_price(self):
```

Comment:
Why this extra test? What is adding different from the previous ones?

Reply:
Codecov was giving me a coverage error for the _get_new_price function (line: return line.price_unit), and I thought it was better to have a test that specifically tested different values of variation_type.